src/opm/common/utility/numeric/RootFinders.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+  Copyright 2010, 2019 SINTEF Digital
+  Copyright 2010, 2019 Equinor ASA
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <config.h>
+#include <opm/common/utility/numeric/RootFinders.hpp>
+
+#include <opm/common/OpmLog/OpmLog.hpp>
+
+#include <fmt/format.h>
+
+namespace Opm
+{
+
+double ThrowOnError::handleBracketingFailure(const double x0, const double x1,
+                                             const double f0, const double f1)
+{
+    const auto str = fmt::format("Error in parameters, zero not bracketed: "
+                                 "[a, b] = [{}, {}]    f(a) = {}    f(b) = {}",
+                                 x0, x1, f0, f1);
+    OpmLog::debug(str);
+    OPM_THROW_NOLOG(std::runtime_error, str);
```

Comment:
It's probably fine to depend on `RootFinders.hpp` including `ErrorMacros.hpp` which in turn includes `<stdexcept>`, but for the sake of making this code a bit more independent we should probably include `<stdexcept>` explicitly here.

On the other hand, it's not entirely clear to me that following this work `RootFinders.hpp` **needs** to include `ErrorMacros.hpp`.

Reply:
it does, there's an OPM_THROW left further down. But I can include my deps in the source, no problem.